src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,57 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final int OPTION = 0;
+    private static final int VALUE = 1;
 
+    public void getStatistic(String fromFileName, String toFileName) {
+        int buy = 0;
```

Comment:
It is necessary to break the logic into 3 methods. The getStatistic() method should only consist of calling these methods:
String dataFromFile = readFile(fromFileName);
String report = createReport(dataFromFile);
writeReportToFile(report, toFileName);